circle.yml

```diff
@@ -5,9 +5,18 @@ machine:
 dependencies:
   override:
     - mvn -U dependency:resolve dependency:resolve-plugins
+
 test:
   override:
     - mvn verify
+  post:
+    - mkdir -p $CIRCLE_TEST_REPORTS/junit/
+    - find . -type f -regex ".*/target/.*-reports/.*xml" -exec cp {} $CIRCLE_TEST_REPORTS/junit/ \;
+    - cp -r target/coverage-reports/jacoco/ $CIRCLE_ARTIFACTS
+    - curl http://www.jpm4j.org/install/script > jpmInstall.sh
```

Comment:
This is the way documented by codacy for circleci. I would like to use their default first. I'm happy to review your PR once this one is merged, though.